### PR TITLE
Industry Julia Users Contributhon follow-up blog post

### PR DESCRIPTION
**Pull request by jrevels**
This is mainly the intro + skeleton for the actual content, which I'll be fleshing out over the course of this week with other Contributhon folks. Wanted to get something up this week to kickstart writing this so I wouldn't fall too far behind on it 😁 

Much of the content will be mined from https://docs.google.com/document/d/1maYTY_hzniSOYdbRiuC1Qqs5H92WydQFUOIaArA-f0g/edit?usp=sharing

### CODE REVIEW

File: blog/2021/01/contributhon.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@def title = "The 2020 Industry Julia Users Contributhon"
+@def authors = "TODO"
+@def published = "TODO"
+@def rss_pubdate = TODO
+@def rss = """A Look Back At The 2020 Industry Julia Users Contributhon"""
+
+A few months ago, I wrote [a blog post](https://julialang.org/blog/2020/09/juliacon-2020-open-source-bof-follow-up/) reflecting on an interesting JuliaCon 2020 discussion between a group of industry-focused Julia users that focused on the interplay between closed-source proprietary software development and Julia's OSS ecosystem. The blog post summarized a few key issues that our discussion group felt served as real hurdles to meaningful OSS contribution from within private organizations. The post concluded by announcing our attempt to jump these hurdles: the Industry Julia Users Contributhon, a hackathon where participating industry organizations could come together to contribute back to the Julia ecosystem.
```
Comment (mattBrzezinski) [inline blog/2021/01/contributhon.md:7]:
```suggestion
A few months ago, I wrote [a blog post](https://julialang.org/blog/2020/09/juliacon-2020-open-source-bof-follow-up/) reflecting on an interesting JuliaCon 2020 discussion between a group of industry-focused Julia users that focused on the interplay between closed-source proprietary software development and Julia's open source software (OSS) ecosystem. 
The blog post summarized a few key issues that our discussion group felt served as real hurdles to meaningful OSS contribution from within private organizations. 
The post concluded by announcing our attempt to jump these hurdles: the Julia Industry Users Contributhon, a hackathon where participating industry organizations could come together to contribute back to the Julia ecosystem.
```

File: blog/2021/01/contributhon.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+This event's goals were to...
+
+> - ...foster/strengthen collaboration across organizational boundaries, and reduce potentially duplicated efforts.
+> - ...both push forward and prove the Julia ecosystem’s readiness for “production” use.
+> - ...provide a nice promotional incentive for the involved organizations to dedicate time to and participate in OSS efforts.
+> - ...provide promotional and technical benefits to the Julia community as whole.
+> - ...have a huge amount of fun!
```
Comment (mattBrzezinski) [inline blog/2021/01/contributhon.md:15]:
```suggestion
> - ...and have a huge amount of fun!
```

File: blog/2021/01/contributhon.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Now, in 2021, I'm happy to report on all of the great contributions made by the wonderful folks at Beacon Biosignals, Invenia, TriScale innov, PumasAI, and RelationalAI (TODO: add links). All shapes and sizes of contribution were welcome at the event: we released whole packages, made PRs, caught bugs, started new projects, and planned out our OSS roadmaps/backlogs for future work. Here's a sampling of the standouts:
```
Comment (mattBrzezinski) [inline blog/2021/01/contributhon.md:17]:
```suggestion
Now, in 2021, I'm happy to report on all of the great contributions made by the wonderful folks at Beacon Biosignals, Invenia, TriScale innov, PumasAI, and RelationalAI (TODO: add links). 
All shapes and sizes of contribution were welcome at the event: we released whole packages, made PRs, caught bugs, started new projects, and planned out our OSS roadmaps/backlogs for future work. 
Here is a sampling of the standouts:
```

**Comment by jrevels** [comment]
Okay, basic content is now here. There were obviously too many contributions to list every single one (and I'm not sure our working doc even captures that list), so I tried to capture the contributions where a single bullet point could convey a complete "narrative" (so mainly ended up listing package contributions/related strings of PRs)

@nhdaly @mattBrzezinski @ffevotte @andreasnoack please review, feel free to add stuff I missed, list yourselves as coauthors if you'd like, etc. :)

File: blog/2021/01/contributhon.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+@def title = "The 2020 Industry Julia Users Contributhon"
+@def authors = "TODO"
```
Comment (mattBrzezinski) [inline blog/2021/01/contributhon.md:2]: Commenting as to not forget these guys

**Comment by mattBrzezinski** [comment]
LGTM! Maybe adding in a point about GatherTown and the social aspecting of putting faces / voices to usernames?

**Review by mattBrzezinski — APPROVED** [review]

**Comment by ffevotte** [comment]
Thanks! This looks very good to me too!

**Review by ffevotte — APPROVED** [review]

File: blog/2021/01/contributhon.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+On top of the contributions themselves, this event taught us about (or in some cases, reminded us of) a few valuable points:
+
+- Julia's package manager works seamlessly even when a single package's registered versions are split across private and public registries. In a lot of cases, this made it super easy to upgrade downstream internal packages to the now-open versions of their upstream dependencies - it just required a version bump!
+- Having a well-documented, battle-tested internal process for safely open-sourcing private code drastically lowers the barrier to actually doing so. Creating an explicit (and if possible, public) "OSS backlog" is a great way to kickstart the development of an internal process.
+- [Gather](https://gather.town/) is actually a pretty useful platform for collaboration! Adding a spacial component to video chat really goes a long way towards recreating the vibe of a shared workspace. After having a great experience using it to host the event, we at Beacon Biosignals created our own internal Gather space that we hang out in daily.
```
Comment (jrevels) [inline blog/2021/01/contributhon.md:47]:
```suggestion
- [Gather](https://gather.town/) is actually a pretty useful platform for collaboration! Just like at JuliaCon, it was fun to meet + hang out with the actual people behind the GitHub handles you come across day-to-day. Also, adding a spacial component to video chat really goes a long way towards recreating the vibe of a shared workspace. After having a great experience using it to host the event, we at Beacon Biosignals created our own internal Gather space that we hang out in daily.
```

in response to @mattBrzezinski 's comment [here](https://github.com/JuliaLang/www.julialang.org/pull/1119#issuecomment-761861093).

**Comment by oxinabox** [comment]
This is good.
May or maynot  want to mention that the event happened in December last year?

File: blog/2021/01/contributhon.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+- [Checkpoints.jl](https://github.com/invenia/Checkpoints.jl), which provides mechanisms for dynamically checkpointing Julia program state (Invenia)
+- [PDMatsExtras.jl](https://github.com/invenia/PDMatsExtras.jl), which extends [PDMats.jl](https://github.com/JuliaStats/PDMats.jl) with a couple different positive (semi-)definite matrix types that [nicely interopate with Distributions.jl](https://github.com/JuliaStats/Distributions.jl/issues/1219) (Invenia)
+- [RegistryCLI.jl](https://github.com/triscale-innov/RegistryCLI.jl), a tool for easily managing private package registries directly from the command line (TriScale innov)
+- [XUnit.jl](https://github.com/RelationalAI-oss/XUnit.jl), a unit test framework with nice parallelization capabilities (RelationalAI)
+- Compiler support for compilation profiling during inference/LLVM optimization, [enabling cool new SnoopCompile.jl features](https://timholy.github.io/SnoopCompile.jl/stable/snoopi_deep/) (RelationalAI)
+- Various quality-of-life improvements to [PProf.jl](https://github.com/JuliaPerf/PProf.jl), including flame graph support. (RelationalAI)
```
Comment (rai-nhdaly) [inline blog/2021/01/contributhon.md:39]:
```suggestion
- Various quality-of-life improvements to [PProf.jl](https://github.com/JuliaPerf/PProf.jl), including interoperability with FlameGraphs.jl and SnoopCompile.jl. (RelationalAI)
```

**Review by rai-nhdaly — COMMENTED** [review]
This is awesome. Thanks for including us even though we missed the actual day-of event 🙈 ❤️

**Comment by jrevels** [comment]
All comments above addressed - thanks for the review! I'll merge tomorrow EOD unless there are any concerns.

**Pull request merged**